YvesIraguha/signature-service
# Feature:Set Up Signature-Service Repository

### What does this PR do? 
- Bootstrap setup for signature-service repository 
- Setup models/controllers/validators/db connection 
- Add tests 

### How can this PR be tested? 
- Clone the repo and checkout the branch `feature/setup-sd-crud` 
- Run `npm install` to install dependencies 
- Run `npm run test` to run the tests, they should all pass. 
- Run `npm run dev` to start the localserver 

### what needs to be considered? 
- ensure that you have environment variables (database connection credentials, and passphrase to use while creating signature algorithms` 
- check the API documentation at `http://localhost:3000/api-docs`, you should be able to send requests to the server. 
- run migrations by running `npm run migrate` to create database tables.
